google-style.yml

```diff
@@ -53,5 +53,9 @@ Style/RescueModifier:
   Enabled: false
 Style/StringLiterals:
   EnforcedStyle: double_quotes
+Style/SymbolArray:
+  EnforcedStyle: brackets
```

Comment:
Is this something we want in the global standard rather than having ruby-docs-samples configure an exception for their usage?

Reply:
I prefer this style. If we all either prefer `[:symbol, :symbol]`, `["string", "string"]` or don't care, we might as well have a uniform approach across repos.